src/jinja2/filters.py

```diff
@@ -273,7 +316,11 @@ def do_xmlattr(
         rv = " " + rv
 
     if eval_ctx.autoescape:
-        rv = Markup(rv)
+        # We don't assume that using this filter we will use a custom
```

Comment:
```suggestion
        # We don't assume that using this filter will use a custom
```

Maybe this way? Sentence made no sense to me.